RecoTracker/TkSeedGenerator/python/DeepCoreSeedGenerator_cfi.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this file should not be needed if you define everything in the fillDescriptions. it will then be created automatically.

Reply:
remove this redundant file completely